src/integrations/prefect-dask/prefect_dask/task_runners.py

```diff
@@ -140,9 +140,7 @@ def result(
             else:
                 return future_result
 
-        _result = self._final_state.result(
-            raise_on_failure=raise_on_failure, fetch=True
-        )
+        _result = self._final_state.result(raise_on_failure=raise_on_failure)
```

Comment:
because of the method removal, this change, and the change in the ray task runner, if this PR gets accepted we may want to use it as a minor version bump

Reply:
nvm reverted this - for another day